modules/ROOT/pages/config-workflow.adoc

```diff
@@ -18,3 +18,7 @@ endif::[]
     * Register Mule runtimes
 
 After completing the steps in this workflow, you can deploy and manage applications, register APIs and use the API Exchange on an Anypoint Platform environment running on your own local network.
+
+[NOTE]
+At the time of this writing, some Docker containers must run as root. Current work in progress removes the need for these containers to run as root. Containers running as root
```

Comment:
processes running as root doesn't expose functionality outside the cluster. All processes serving traffic to outside the cluster run as non-root, (reducing the attack surface)?

Reply:
@federicoboerr PTAL at latest updates - thank you!